Update operator-airgapped.md (#1028)
wandb/docs: content/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

- [W&B Platform](https://github.com/wandb/helm-charts/tree/main/charts/operator-wandb)

> The `operator` chart is used to deploy the W&B Operator aka the Controller Manager. While the `platform` chart is used to deploy the W&B Platform using the values configured in the custom resource definition (CRD).

Reviewer: ```suggestion
The `operator` chart is used to deploy the W&B Operator, which is also referred to as the Controller Manager. The `platform` chart is used to deploy the W&B Platform using the values configured in the custom resource definition (CRD).
```
Fixing a sentence fragment

> 

Author: `aka` is subject to review :)

Reviewer: Are we waiting on anyone else to weigh in?

```

Replace the tag with the version that is available in your internal registry.

> Execute the following to install the operator and the CRD:

Reviewer: ```suggestion

Install the operator and the CRD:
```

```bash
helm upgrade --install operator wandb/operator -n wandb --create-namespace -f values.yaml
```

> You can find all supported values in the [official Kubernetes operator repository](https://github.com/wandb/helm-charts/blob/main/charts/operator/values.yaml).

Reviewer: ```suggestion
For full details about the supported values, refer to the [Kubernetes operator GitHub repository](https://github.com/wandb/helm-charts/blob/main/charts/operator/values.yaml).
```

## Step 6: Configure W&B Custom Resource 

After installing the W&B Kubernetes operator, you must configure the Custom Resource (CR) to point to your internal Helm repository and container registry.

This configuration ensures that the Kubernetes operators uses your internal registry and repository are when it deploys the required components of the W&B platform. 

> Save this example CR as a file named `wandb.yaml`.

Reviewer: ```suggestion
Copy this example CR to a new file named `wandb.yaml`.
```

    
```

> To deploy the W&B platform, the Kubernetes Operator uses the `operator-wandb` chart from your internal repository and use the values from your CR to configure the Helm chart.

Reviewer: ```suggestion
To deploy the W&B platform, the Kubernetes Operator uses the values from your CR to configure the `operator-wandb` Helm chart from your internal repository.
```

Replace all tags/versions with the versions that are available in your internal registry.

More information on creating the preceding configuration file can be found [here]({{< relref "../kubernetes-operator/#configuration-reference-for-wb-server" >}}). 

## Step 7: Deploy the W&B platform

> Finally, after setting up the Kubernetes operator and the CR, deploy the W&B platform using the following command:

Reviewer: ```suggestion
Now that the Kubernetes operator and the CR are configured, apply the `wandb.yaml` configuration to deploy the W&B platform:
```

You can turn off auto-updates from the W&B console. Reach out to your W&B team for any questions on the supported versions. Also, note that W&B supports platform versions released in last 6 months. W&B recommends performing periodic upgrades. 

### Does the deployment work if the environment has no connection to public repositories?

> As long as you have enabled the `airgapped: true` configuration, the Kubernetes operator does not attempt to reach public repositories. The Kubernetes operator attempts to use your internal resources.

Reviewer: I think another PR in flight updates this sentence and I gave a suggested edit. Please keep this in sync.
